pkg/services/serviceaccounts/serviceaccounttest/fake.go

```diff
@@ -0,0 +1,79 @@
+package serviceaccounttest
+
+import (
+	"context"
+
+	"github.com/grafana/grafana/pkg/services/apikey"
+	"github.com/grafana/grafana/pkg/services/serviceaccounts"
+)
+
+// Service implements the API exposed methods for service accounts.
+type service interface {
```

Comment:
Sorry, I do not understand this pattern. We have a type `service` in api.go, but what a purpose of duplication? We can export type from api.go (by making it capital case, `type Service interface {`. 

And it's not related to this PR, but I do not understand a reason for exposing all API methods as a service. If we use some methods, like `HideApiKeysTab()` outside of api package, we can add this method to service implementation.